src/components/login.tsx

```diff
@@ -112,12 +117,26 @@ export class LoginScreen extends React.Component<LoginProps, LoginState> {
           />
         </View>
 
+        <View style={Styles.switchContainer}>
+          <Switch
+            value={this.state.rememberMe}
+            onValueChange={(value) => this.setState({rememberMe: value})}
```

Comment:
looks like rememberMe is defaulting to false above when we actually make the call to login.  please base it on state.

Reply:
Sorry, didn't get this comment. The default value is `false` and if the switch is not changed, it will be the value used in the request. The `Switch` uses the value from the state `value={this.state.rememberMe}` and sets a new value in the state when the Switch has been changed `onValueChange={(value) => this.setState({rememberMe: value})}`